### ISSUE TITLE
Error on image upload through admin site 

### ISSUE THREAD

**Issue by danconte72** [meumobi/sitebuilder#644]
### Expected behaviour
image properly uploaded 

### Actual behaviour
![image](https://user-images.githubusercontent.com/32454870/53983800-d0292280-40f6-11e9-990d-1b26de77c9e2.png)

### Steps to reproduce
1. create a new item
2. upload a image

### Expected responses
- Why it happens
- How to fix it
- How to test

**Comment by danconte72** [comment]
on log
```log
[2019-03-07 20:31:22] sitebuilder.ERROR: "uploadFile failed" {"model":{"order":0,"created":"[object] (MongoDate: 0.28400000 1551987082)","modified":"[object] (MongoDate: 0.28400000 1551987082)","published":"[object] (MongoDate: 0.28400000 1551987082)","is_published":true,"thumbnails":[],"groups":[],"notification_id":""},"path":"uploads/items","message":"upload folder is not writeable","exception":"[object] (Exception(code: 0): upload folder is not writeable at /home/meumobi/PROJECTS/meumobi.com/releases/20181119114456/sitebuilder/lib/utils/FileUpload.php:87)","component":"images"}
[2019-03-07 20:33:03] sitebuilder.ERROR: "uploadFile failed" {"model":{"order":0,"created":"[object] (MongoDate: 0.10500000 1551987183)","modified":"[object] (MongoDate: 0.10500000 1551987183)","published":"[object] (MongoDate: 0.10500000 1551987183)","is_published":true,"thumbnails":[],"groups":[],"notification_id":""},"path":"uploads/items","message":"upload folder is not writeable","exception":"[object] (Exception(code: 0): upload folder is not writeable at /home/meumobi/PROJECTS/meumobi.com/releases/20181119114456/sitebuilder/lib/utils/FileUpload.php:87)","component":"images"}
[2019-03-07 20:33:49] sitebuilder.ERROR: "uploadFile failed" {"model":{"order":0,"created":"[object] (MongoDate: 0.20700000 1551987229)","modified":"[object] (MongoDate: 0.20700000 1551987229)","published":"[object] (MongoDate: 0.20700000 1551987229)","is_published":true,"thumbnails":[],"groups":[],"notification_id":""},"path":"uploads/items","message":"upload folder is not writeable","exception":"[object] (Exception(code: 0): upload folder is not writeable at /home/meumobi/PROJECTS/meumobi.com/releases/20181119114456/sitebuilder/lib/utils/FileUpload.php:87)","component":"images"}
```

no writable folder

**Comment by danconte72** [comment]
When I've performed img- backups routine as root, the script has changed the folder mod.
Sou I've update `$ chmod -R 777 /shared/uploads/items` in order to allow (meumobi user) write on folder properly

**Comment by vdias38** [comment]
ok, but the source of issue is not solved: **If somebody run the img- backups routine as root the script change the folder mod**
Then we have 2 options:

- add a console log warning to not use su to run script
- at the end of script change folder mod

**Comment by vdias38** [comment]
@danconte72 is there stg we should do on this issue ? if not can I close it ?

**Comment by danconte72** [comment]
error will be fixed on #646
